views/js/testRunner/ResumingStrategy/resetAfterResume.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2015 (original work) Open Assessment Technologies SA;
+ *
+ * Contain dummy implementation
+ */
+
+define([], function () {
+    'use strict';
+
+    var sessionStateFactory = function sessionStateFactory() {
+
+        /**
+         */
+        var sessionState = {
+
+            getDuration: function getDuration() {
+                return 0;
+            },
+
+            restart: function restart() {
+            }
+
```

Comment:
This code does not erase the local storage, so there may be still a value inside after reset. Is it intentional ?